auxx/src/Command/Update.hs

```diff
@@ -127,10 +131,34 @@ propose sendActions ProposeUpdateParams{..} = do
             if null ccPeers
                 then putText "Error: no addresses specified"
                 else do
-                    submitUpdateProposal (immediateConcurrentConversations sendActions ccPeers) ss updateProposal
-                    let id = hash updateProposal
-                    putText $
-                      sformat ("Update proposal submitted, upId: "%hashHexF) id
+                    let upid = hash updateProposal
+                    let enqueue = immediateConcurrentConversations sendActions ccPeers
+                    if not voteAll then do
+                        submitUpdateProposal enqueue ss updateProposal
+                        putText $ sformat ("Update proposal submitted, upId: "%hashHexF) upid
+                    else do
+                        votes <- constructVotes upid
+                        sendUpdateProposal enqueue upid updateProposal votes
+                        putText $ sformat ("Update proposal submitted along with votes, upId: "%hashHexF) upid
+  where
+    constructVotes :: UpId -> AuxxMode [UpdateVote]
+    constructVotes upid = do
+        skeys <- getSecretKeysPlain
+        signatures <-
+            withSafeSigners
+                skeys
+                (pure emptyPassphrase)
+                (pure . map (\s -> safeSign SignUSVote s (upid, True)))
+        let safeZip a b =
+                if length a == length b then zip a b
+                else error $ "Number of signatures: " <> show (length b) <> ", expected " <> show (length a)
+        forM (safeZip skeys signatures) $ \(skey, signature) -> pure $
+            UpdateVote
+                { uvKey        = encToPublic skey
+                , uvProposalId = upid
+                , uvDecision   = True
+                , uvSignature  = signature
+                }
```

Comment:
I think the code would be cleaner if you changed the second argument of `submitUpdateProposal` to `[SafeSigner]` (i. e. generalize it to multiple signers) and implemented the `not voteAll` case using that generalized version.